dbms/src/Flash/tests/bench_aggregation_hash_map.cpp

```diff
@@ -0,0 +1,331 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <AggregateFunctions/registerAggregateFunctions.h>
+#include <Columns/ColumnDecimal.h>
+#include <DataTypes/DataTypeDate.h>
+#include <DataTypes/DataTypeDecimal.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Flash/Coprocessor/AggregationInterpreterHelper.h>
+#include <Flash/Coprocessor/DAGCodec.h>
+#include <Flash/Coprocessor/DAGExpressionAnalyzer.h>
+#include <Flash/Planner/PhysicalPlanHelper.h>
+#include <Interpreters/Aggregator.h>
+#include <Interpreters/Context.h>
+#include <TestUtils/TiFlashTestBasic.h>
+#include <TestUtils/TiFlashTestEnv.h>
+#include <benchmark/benchmark.h>
+#include <google/protobuf/util/json_util.h>
+#include <tipb/executor.pb.h>
+
+#include <random>
+#include <string>
+
+namespace DB
+{
+namespace tests
```

Comment:
Please also add unit tests in https://github.com/pingcap/tiflash/blob/8e50de84e6d6ecdcc108990217b70b6bb3f50271/dbms/src/Flash/tests/gtest_aggregation_executor.cpp